.github/workflows/tests.yml

```diff
@@ -0,0 +1,91 @@
+name: Tests
+on:
+  pull_request:
+  push:
+    branches:
+    - gha
```

Comment:
Disregard the comment about the `gha` branch, since it has been removed.

Reply:
You are correct although I will need to briefly restore testing on the `gha` branch when I make changes to this PR so that I can make sure everything still works